integration-tests/test_207.py

```diff
@@ -0,0 +1,28 @@
+import os
+import sys
+
+try:
+    import numpy as np
```

Comment:
I think better to add numpy to test dependencies https://github.com/RaRe-Technologies/smart_open/blob/master/setup.py#L19

Reply:
Or even add it like standart unittest (not integration), wdyt?

Reply:
TBH, I don't think numpy belongs in our dependencies, because we only need it for one thing: to run this test.

What else would be the benefit of including it?

Reply:
> because we only need it for one thing: to run this test.

yes, and I'm talking about test dependencies (not requirements of `smart_open` package)

> What else would be the benefit of including it?

No, only for correct run test, that's all